AStarAlgorithmsProject/Driver/MainDriver.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace AStarAlgorithmsProject
+{
+    /// <summary>
+    /// Class for manipulating the funtionality of the program. Such as updating the grid we are pathing through, and passing it to the algorithim 
+    /// </summary>
+    class MainDriver
+    {
+        AStarSolver aSS; // I know what I did. 
+        Point[,] points; // The cordinates for whatever interactable map we decide on.
+
+        public MainDriver(int size = 10) // default constructor. Currently, design does not support resizing the map during run time
+        {
+            aSS = new AStarSolver(size);
+            points = new Point[size, size];
+
+            for (int i = 0; i < points.GetLength(0); i++)
+            {
+                for (int j = 0; j < points.GetLength(1); j++)
+                {
+                    points[i, j] = new Point(i, j);
+                }
+            }
+
+            // hard coding impassable terrain - testing use only
+            aSS.SetPassalbe(new Point(8, 8), false);
+            aSS.SetPassalbe(new Point(8, 9), false);
+          //  aSS.SetPassalbe(new Point(9, 8), false); //Uncomment to see a map where no solution exsists based on the start and end in PrintMap
+        }
+
+        //
+        // Methods for, like, changing what points are blocked or their cost and so on.
+        //
+
+        // Test method for displaying the map and the path A* tool to get the the end point.
+        public string PrintMap()
```

Comment:
Kinda misleading name. This method also creates a solver and solves it. Might be best to have a differant method to print a map that has been used to get a solution.

Reply:
This is just a test method to quickly make a solver and get a result. Name changed to reflect that.